pulp_smash/tests/docker/api_v2/test_crud.py

```diff
@@ -15,6 +15,11 @@
 from pulp_smash.constants import REPOSITORY_PATH
 
 
+def setUpModule():  # pylint:disable=invalid-name
```

Comment:
It might be possible to define this once in another module and `import set_up_module as setUpModule`.